scavenge/scavenge/type.diff

```diff
@@ -1,18 +1,18 @@
-diff --git a/starport-scavenge/scavenge/vue/src/store/app.js b/starport-scavenge/scavenge/vue/src/store/app.js
+diff --git a/scavenge/scavenge/vue/src/store/app.js b/scavenge/scavenge/vue/src/store/app.js
```

Comment:
never seen this file before. do we need it?

Reply:
Removed it!